lib/package.js

```diff
@@ -473,7 +475,7 @@ exports.download = function(pkg, options, installDeps) {
         if (endpoint.getOverride) {
           if (!override && config.pjson.overrides[pkg.exactName])
             ui.log('warn', 'Using local override for `' + pkg.exactName + '`');
-          return endpoint.getOverride(pkg.registry, pkg.package, pkg.version, override || config.pjson.overrides[pkg.exactName]);
```

Comment:
Surely this line already does everything we want it to do though? How does the new `overrides` object provide more information?